Example/ChatExample.xcodeproj/project.pbxproj

```diff
@@ -661,10 +669,10 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				DEVELOPMENT_TEAM = 9EUCWD26TP;
+				DEVELOPMENT_TEAM = 28VWFVMBDR;
 				INFOPLIST_FILE = "$(SRCROOT)/Sources/Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = com.messagekit.ChatExample;
+				PRODUCT_BUNDLE_IDENTIFIER = com.messagekit.ChatExample.123;
```

Comment:
Mind changing this back to `com.messagekit.ChatExample`?

Reply:
Sorry it’s a typo! Fixing ASAP